### PR TITLE
[VGAFONTEDIT] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to PR https://github.com/reactos/reactos/pull/5908

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: modules/rosapps/applications/devutils/vgafontedit/lang/ro-RO.rc [hunk 1]
```diff
@@ -1,10 +1,11 @@
 /*
  * PROJECT:     ReactOS VGA Font Editor
- * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
  * PURPOSE:     Romanian resource file
- * COPYRIGHT:   Copyright 2008 Colin Finck (colin@reactos.org)
- * TRANSLATOR:  Ștefan Fulea (stefan dot fulea at mail dot com)
+ * TRANSLATORS: Copyright 2008 Ștefan Fulea  <stefan.fulea@mail.com>
```
Comment (binarymaster) [inline modules/rosapps/applications/devutils/vgafontedit/lang/ro-RO.rc:5]:
```suggestion
 * TRANSLATORS: Copyright 2008 Ștefan Fulea <stefan.fulea@mail.com>
```
(excess space)

File: modules/rosapps/applications/devutils/vgafontedit/lang/ro-RO.rc [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+ *              Copyright 2023 Andrei Miloiu <miloiuandrei@gmail.com>
  */
+
```
Comment (binarymaster) [inline modules/rosapps/applications/devutils/vgafontedit/lang/ro-RO.rc:8]:
```suggestion
```
(excess line)

File: modules/rosapps/applications/devutils/vgafontedit/lang/ro-RO.rc [hunk 3]
```diff
@@ -1,10 +1,11 @@
 /*
  * PROJECT:     ReactOS VGA Font Editor
- * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
```
Comment (binarymaster) [inline modules/rosapps/applications/devutils/vgafontedit/lang/ro-RO.rc:3]:
```suggestion
 * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
```
(keep this as is)

**Review by JoachimHenze — APPROVED** [review]

**Pull request merged**